src/components/profile/CircleButtonWrapper.js

```diff
@@ -1,24 +1,42 @@
 // @flow
 import React from 'react'
-import { TouchableOpacity } from 'react-native'
+import { TouchableOpacity, View } from 'react-native'
 import { withStyles } from '../../lib/styles'
-import { Icon } from '../common'
+import { Icon, Text } from '../common'
 
-const CircleButtonWrapper = props => {
-  const { onPress, disabled, styles, style, iconName, iconColor = '#fff', iconSize = 20 } = props
-
-  return (
+const CircleButtonWrapper = ({
+  label,
+  labelStyles,
+  onPress,
+  disabled,
+  styles,
+  style,
+  containerStyle,
+  iconName,
+  iconColor = '#fff',
```

Comment:
Please don't use default values in the destructurized props list.
You should go React-way and use .defaultProps property designed especially for specifying defaults.

Reply:
done